packages/medusa/src/services/shipping-option.ts

```diff
@@ -397,6 +397,31 @@ class ShippingOptionService extends TransactionBaseService {
     return option
   }
 
+  private async validateAndSetOptionPrice(
```

Comment:
Question: here we are both setting the value to the options object reference and returning it. I think the name is pretty clear about the fact that it mutate the input object. Should we not return it then?

Reply:
Makes sense. I'd rather have us not mutate it directly and instead return it. Will add that 💪